default.nix

```diff
@@ -5,6 +5,14 @@ python3.pkgs.buildPythonApplication rec {
   name = "nixpkgs-review";
   src = ./.;
   buildInputs = [ makeWrapper ];
+
+  propagatedBuildInputs = [
+    python3.pkgs.humanize
+    # humanize fails to declare its dependency on septools correctly
+    # https://github.com/NixOS/nixpkgs/pull/113060
```

Comment:
That PR is merged.

Reply:
Is that sufficient? I personally call this default.nix with nixpkgs from 20.09.

Reply:
We can backport this is required.

Reply:
Would you be able to do that? I've never done a backport.

Reply:
https://github.com/NixOS/nixpkgs/pull/118153